src/devices/bus/spectrum/mface.cpp

```diff
@@ -172,7 +172,9 @@ ROM_START(mface128)
 	ROM_REGION(0x2000, "rom", 0)
 	ROM_DEFAULT_BIOS("v363c")
 	ROM_SYSTEM_BIOS(0, "v363c", "87.2 V36 3C")
-	ROMX_LOAD("mf128_36_3c.rom", 0x0000, 0x2000, CRC(78ec8cfd) SHA1(8df204ab490b87c389971ce0c7fb5f9cbd281f14), ROM_BIOS(0))
+	ROMX_LOAD("mf128_36_3c.rom", 0x0000, 0x2000, CRC(78ec8cfd) SHA1(8df204ab490b87c389971ce0c7fb5f9cbd281f14), ROM_BIOS(0))	
```

Comment:
Why does this line show as changed?  I’m always wary of invisible changes to lines.

Reply:
I was considering re-sorting the ROMs (putting the new one over the old one) but then decided against it.

Reply:
If there’s no net change to the line, it shouldn’t show as changed in the diff.  What’s actually changed in this line to cause it to be highlighted in the diff?

Reply:
perhaps was added trailing space ?

on the side note - why not add 2 more remaining MF128 dumps listed there ? https://x128.speccy.cz/multiface/multiface.htm

Reply:
@p1pkin a trailing space it was, thanks! You led me down quite a rabbit hole with that link ... the actual ROMs are found elsewhere.
@cuavas building SRCCLEAN right now ,sorry ... :)